src/services/member/plugins/action/schemas.ts

```diff
@@ -4,3 +4,14 @@ import { idParam } from '../../../../schemas/fluent-schema';
 export const deleteAllById = {
   params: idParam,
 };
+
+export const getMemberFilteredActions = {
+  querystring: {
+    type: 'object',
+    properties: {
+      startDate: { type: 'string' },
+      endDate: { type: 'string' },
```

Comment:
Not sure it is possible, but do you think you can provide some kind of format? since it's supposed to be dates.